src/index.html

```diff
@@ -8,6 +8,77 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="frame">
+      <div class="container">
+        <div class="stars stars--0">
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+
+          <div class="stars_star"></div>
+        </div>
+        <div class="stars stars--1">
```

Comment:
```suggestion
        <div class="stars stars--0">
          <div class="stars_star"></div>
          <div class="stars_star"></div>
          <div class="stars_star"></div>
          <div class="stars_star"></div>
          <div class="stars_star"></div>
        </div>
        
        <div class="stars stars--1">
```
you should add spaces between multiline sibling blocks, no need to add them between one-line elements